docs/Roadmap.md

```diff
@@ -0,0 +1,122 @@
+# Proxy-Wasm Roadmap
+
+The Proxy-Wasm community and maintainers envision an evolution path that has the
+following tracks:
+
+*   [Spec / ABI](#abi)
+*   [SDKs / language support](#sdks)
+*   [Host features](#host)
+*   [Envoy integration](#envoy)
+
+Each track is described in more detail below, with owners and ETAs listed for
+efforts currently in development. This roadmap should not be construed as a set
+of commitments, but rather a set of directions that are subject to change in
+response to community interest and contributions.
+
+The overarching goals of this document are to:
+
+*   Publish areas of current investment.
+*   Encourage external contributors by pointing out feature gaps.
+*   Align this repository with the vision of WebAssembly: a portable technology
+    that is cross-language, cross-platform, and cross-provider.
+
+<a name="abi"></a>
+
+## Spec / ABI
+
+*   (Q1'25: @piotrsikora, @mpwarres) Publish ABI v0.3, containing at least:
+    *   Feature negotiation (proxy-wasm/spec#71 and proxy-wasm/spec#56)
+    *   Better header/body buffering support (proxy-wasm/spec#63)
+    *   Async shared data (proxy-wasm/spec#54)
+    *   Repeated header support (proxy-wasm/spec#53)
+*   (Help wanted) WASI convergence. We want to adopt the component model at WASI
+    1.0. There is a lot of overlap between Proxy-Wasm and some WASI proposals
+    ([wasi-http](https://github.com/WebAssembly/wasi-http),
+    [wasi-keyvalue](https://github.com/WebAssembly/wasi-keyvalue), etc). In the
+    short term, we'd like to define the Proxy-Wasm ABI in
+    [WIT](https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md),
+    to understand:
+    *   How do
+        [Proxy-Wasm interfaces](https://github.com/proxy-wasm/proxy-wasm-cpp-host/blob/main/include/proxy-wasm/context_interface.h)
+        map to components?
```

Comment:
The `ContextInterface` is an interface between Proxy-Wasm C++ Host and proxies embedding it, it's not a Proxy-Wasm interface.